pyproject.toml

```diff
@@ -84,7 +84,7 @@ tsfresh = [
     "featuretools-tsfresh-primitives >= 1.0.0",
 ]
 premium = [
-    "premium_primitives >= 0.0.2",
+    "premium_primitives @ git+https://github.com/alteryx/premium_primitives.git@main",
```

Comment:
Once premium primitives 0.0.3 is available on pypi, we should be able to go back to requiring `>= 0.0.2`